app/javascript/dashboard/store/modules/contacts/actions.js

```diff
@@ -169,4 +169,14 @@ export const actions = {
       root: true,
     });
   },
+
+  updateContact: async ({ commit }, updateObj) => {
+    commit(types.SET_CONTACT_UI_FLAG, { isUpdating: true });
+    try {
+      commit(types.EDIT_CONTACT, updateObj);
+      commit(types.SET_CONTACT_UI_FLAG, { isUpdating: false });
+    } catch (error) {
+      commit(types.SET_CONTACT_UI_FLAG, { isUpdating: false });
```

Comment:
Throw error here so that error cases can be handled at Vue side with try catch there.

Reply:
@nithindavid The `updateContact` method is actually calling from action cable event (https://github.com/chatwoot/chatwoot/pull/3334/files#diff-0deb2706ff6e98c1ca6f8864a01297cdb06131d958c2f7b506ba3d8a99d64989R129). We are not handling any errors related action cable event update.